### ISSUE TITLE
Maybe you should do...

### ISSUE THREAD

**Issue by tehroxie** [SpongyBacon/spongybacon.com#4]
As of now, the webiste looks cool and good... but.... its very bootstrappy ya know? its easy to tell ur using bootstrap because you havent really changed much... im a web designer and i luv flat design and fllscreen landing pages. i think you should invest more time on those particular two things. if you need or want any help, im still alive for now. :D

**Comment by SpongyBacon** [comment]
Yeah, I didn't really work on the styling. Planning on remaking it with https://github.com/google/material-design-lite soon :P

**Comment by SpongyBacon** [comment]
Didn't mean 2 close
